src/ui/find-ui/src/FamilyHubs.ServiceDirectory.Web/Models/Service.cs

```diff
@@ -10,17 +10,14 @@ public enum ServiceType
     //todo: type hierarchy, rather than type? or just null what we don't have?
     // when / opening hours will show regular schedule only. holiday schedule will be ignored for mvp (probably just show the description field)
     public sealed record Service(
```

Comment:
Do we compare Service instances? Just wondering if a record is appropriate in this instance given the extra comparators etc that get added?

Reply:
Not sure, I think it's best to leave it as it's been modified as part of other PRs too and would increase scope of this one, + likely during the single directory stuff is that we will just re-evaluate everything anyway (since Find will be removed)

Reply:
I think I had to add extra comparison methods in my test PR as the default ones for record don't work. And obviously that will need changing after this.